packages/botonic-cli/src/botonic-api-service.ts

```diff
@@ -225,79 +242,83 @@ export class BotonicAPIService {
     orgName: string,
     campaign: any
   ): Promise<any> {
-    const url = `${this.baseApiUrl}users/`
     const signupData = { email, password, org_name: orgName, campaign }
-    return axios({ method: 'post', url: url, data: signupData })
+    return this.apiPost({ path: '/v1/users/', body: signupData })
   }
 
   async saveBot(botName: string): Promise<AxiosPromise> {
```

Comment:
createBot

Reply:
Changed in the next [PR#2894](https://github.com/hubtype/botonic/pull/2894)